src/common/style/derived-css-vars.ts

```diff
@@ -0,0 +1,30 @@
+import { css } from "lit";
+
+const trimWithLineBreak = (str: string) => str.replaceAll("\n", "").trim();
+
+export const extractDerivedVars = (
+  string: TemplateStringsArray,
+  ...values: any[]
+) => {
+  const cssString = string.reduce(
```

Comment:
```suggestion
  strings: TemplateStringsArray,
  ...values: any[]
) => {
  const cssString = strings.reduce(
```
Calling it `string` is confusing

Reply:
thx, I changed it completely